Js/model.js

```diff
@@ -0,0 +1,104 @@
+const projectBtn = document.querySelectorAll('.projectBtn');
+const body = document.querySelector('body');
+const imagesSrc = ['./images/project1.png', './images/project5.png', './images/project2.png', './images/project4.png', './images/project3.png', './images/project6.png'];
+const techTags = ['HTML/CSS', 'Ruby on Rails', 'JavaScript'];
+const work = document.querySelector('.work');
+projectBtn.forEach((element, index) => {
+  /* eslint-disable no-unused-vars */
+  element.addEventListener('click', (event) => {
+    body.classList.add('active');
+
+    const container = document.createElement('div');
+    container.className = 'container-base';
+    work.prepend(container);
+
+    const pop = document.createElement('div');
+    pop.className = 'container-popup';
+    container.appendChild(pop);
+
+    const popClose = document.createElement('button');
+    popClose.className = 'popup-close';
+    popClose.innerText = 'X';
+    /* eslint-disable no-unused-vars */
+    popClose.addEventListener('click', ((event) => {
+      body.classList.remove('active');
+      container.remove();
+    }));
+    pop.appendChild(popClose);
+
+    const popTitle = document.createElement('h2');
+    popTitle.innerHTML = 'Project name goes here';
+    popTitle.className = 'pop-title';
+    pop.appendChild(popTitle);
+
+    const popTechList = document.createElement('ul');
+    popTechList.className = 'pop-tech-list';
+    pop.appendChild(popTechList);
+
+    techTags.forEach((element) => {
+      const popTechListContent = document.createElement('li');
+      popTechListContent.innerHTML = element;
+      popTechList.appendChild(popTechListContent);
+    });
+
+    const popImage = document.createElement('img');
+    popImage.src = imagesSrc[index];
+    popImage.className = 'pop-img';
+    pop.appendChild(popImage);
+
+    const popImageContainer = document.createElement('div');
+    popImageContainer.className = 'pop-image-container';
+    pop.appendChild(popImageContainer);
+
+    /* eslint-disable no-plusplus */
```

Comment:
- [ ] Please remove the **eslint disable** statement. Because ` Eslint` helps developers to write clean JavaScript codes and adapt best practices when writing the code.  You just need to export it as the default code. :+1: